src/single-spa.js

```diff
@@ -1,556 +1,18 @@
-import CustomEvent from 'custom-event';
-import { handleChildAppError } from './single-spa-child-app-error.js';
-import { ensureJQuerySupport } from './jquery-support.js';
-import { parseUri } from './single-spa.helpers.js';
-
-const window = typeof window !== 'undefined' ? window : (typeof global !== 'undefined' ? global : self);
-
-const originalAddEventListener = window.addEventListener;
```

Comment:
All the deleted code in this file got cut / pasted into other files. One of the big things with 3.0 is that single-spa.js was split up into many files.